rca/enquire_to_study/templates/enquire_to_study/index.html

```diff
@@ -15,92 +15,53 @@
-                        {% if view.has_filters and all_count %}
-                        <div class="changelist-filter col3">
-                            <h2>{% trans 'Filter' %}</h2>
-                            {% for spec in view.filter_specs %}{% admin_list_filter view spec %}{% endfor %}
-                        </div>
-                        {% endif %}
-                    {% endblock %}
-
-                    <div class="result-list {% if view.has_filters and all_count %}col9{% else %}col12{% endif %}">
-                        {% block result_list %}
-                            {% if not all_count %}
-                                <div class="nice-padding" style="margin-top:30px;">
-                                    {% if no_valid_parents %}
-                                        <p>{% blocktrans with view.verbose_name_plural as name %}No {{ name }} have been created yet. One of the following must be created before you can add any {{ name }}:{% endblocktrans %}</p>
-                                        <ul>
-                                            {% for type in required_parent_types %}<li><b>{{ type|title }}</b></li>{% endfor %}
-                                        </ul>
-                                    {% else %}
-                                        <p>{% blocktrans with view.verbose_name_plural as name %}No {{ name }} have been created yet.{% endblocktrans %}
-                                        {% if user_can_create %}
-                                            {% blocktrans with view.create_url as url %}
-                                                Why not <a href="{{ url }}">add one</a>?
-                                            {% endblocktrans %}
-                                        {% endif %}</p>
-                                    {% endif %}
-                                </div>
-                            {% else %}
-                                {% result_list %}
-                            {% endif %}
-                        {% endblock %}
-                    </div>
-
-                    {% block pagination %}
-                        <nav class="pagination {% if view.has_filters and all_count %}col9{% else %}col12{% endif %}" aria-label="{% trans 'Pagination' %}">
-                            <p>{% blocktrans with page_obj.number as current_page and paginator.num_pages as num_pages %}Page {{ current_page }} of {{ num_pages }}.{% endblocktrans %}</p>
-                            {% if paginator.num_pages > 1 %}
-                                <ul>
-                                    {% pagination_link_previous page_obj view %}
-                                    {% pagination_link_next page_obj view %}
-                                </ul>
-                            {% endif %}
-                        </nav>
-                    {% endblock %}
-
-                {% endblock %}
-            </div>
-        </div>
+        {{ block.super }}
```

Comment:
Since there are no significant changes to this block, I have just edited this to use the "super" in case the future wagtail releases change the structure of the main content.